src/components/advanced-table/advanced-table.tsx

```diff
@@ -77,7 +77,10 @@ export function AdvancedTable<
   const [page, setPage] = useState<number>(0);
   const [maxPage, setMaxPage] = useState<number>(0);
   const [loading, setLoading] = useState<boolean>(true);
-  const [displayError, setDisplayError] = useState<string | null>(null);
+  const [displayError, setDisplayError] = useState<React.ReactNode | null>(
+    null
+  );
+  const [displayInfo, setDisplayInfo] = useState<React.ReactNode | null>(null);
```

Comment:
`setDisplayInfo()` is never called and thus this state variable is never used meaningfully. It looks like the only significant change is that the display error is dismissible. Could you revert all the other changes in this file?